polylogue/html.py

```
@@ -393,3 +559,36 @@ def repl(match: re.Match[str]) -> str:
         return "\n".join(parts)
 
     return pattern.sub(repl, html)
+
+
+_IMAGE_SUFFIXES = {".png", ".jpg", ".jpeg", ".gif", ".webp", ".bmp", ".svg"}
+
+
+def _attachment_preview_src(att: AttachmentInfo) -> Optional[str]:
```

Ruff (0.15.9)

[error] Undefined name `Optional`

(F821)